src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String LINE = "    __________________________________________________________________________";
+    private static final String INDENT = "     ";
+
+    private static boolean isClosed = false;
+    private static TaskList tasks = new TaskList();
+    private static int counter = 0;
```

Comment:
Perhaps this variable could be removed since it is not used anywhere else?

Reply:
Cool, I'll consider removing it for the next iterations.  
Thank you for your comments, I really appreciate them. I'll take them into consideration and implement the changes in the next iteration :smile: